### PR TITLE
Add Python resources to Installation tutorial

### LINKED ISSUE #24: Docs page - how to install python/conda/napari from scratch
We need a documentation page explaining how to install python/conda/napari from scratch (also jupyter notebooks?)

Good links:
* The [Anaconda distribution](https://www.anaconda.com/distribution/)
of python is recommended.
* [Managing conda environments](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
* [Try Jupyter](https://jupyter.org/try)

### PR DESCRIPTION
**Pull request by melissawm**
# References and relevant issues
Closes #24

# Description
Adds additional resources about installing python and setting up virtual environments to the Installation tutorial.

### CODE REVIEW

File: docs/tutorials/fundamentals/installation.md [hunk 1]
```diff
@@ -41,6 +39,12 @@ You may also want:
 - an environment manager like [conda](https://docs.conda.io/en/latest/miniconda.html) or
 [venv](https://docs.python.org/3/library/venv.html) **(Highly recommended)**
 
+```{note}
+New to Python or not familiar with virtual environments? You can use our
```
Comment (psobolewskiPhD) [inline docs/tutorials/fundamentals/installation.md]:
```suggestion
New to Python or uncertain about conda, pip, and virtual environments? You can use our
```

Totally optional, but trying to make it more explicit in terms of terminology people could have heard about and we use below.

**Review by psobolewskiPhD — APPROVED** [review]
This is great -- and I totally dropped the ball on doing this.
What do you think of putting the 3 links in the admonition at the top? Would it be too much?

**Comment by melissawm** [comment]
How about this version, @psobolewskiPhD ?

**Review by psobolewskiPhD — APPROVED** [review]
Love it!

**Pull request merged**